src/Tokenizer/Transformer/ImportTransformer.php

```diff
@@ -51,12 +51,17 @@ public function process(Tokens $tokens, Token $token, int $index): void
 
         $prevToken = $tokens[$tokens->getPrevMeaningfulToken($index)];
 
-        if ($prevToken->isGivenKind(T_USE)) {
-            $tokens[$index] = new Token([
-                $token->isGivenKind(T_FUNCTION) ? CT::T_FUNCTION_IMPORT : CT::T_CONST_IMPORT,
-                $token->getContent(),
-            ]);
+        if (!$prevToken->isGivenKind(T_USE)) {
```

Comment:
```php
    /**
     * {@inheritdoc}
     */
    public function getPriority(): int
    {
        // Should run after CurlyBraceTransformer and ReturnRefTransformer
        return -1;
    }
```

```php
        if (!$prevToken->isGivenKind(T_USE)) {
            $nextToken = $tokens[$tokens->getNextTokenOfKind($index, ['=', '(', [CT::T_RETURN_REF], [CT::T_GROUP_IMPORT_BRACE_CLOSE]])];

            if (!$nextToken->isGivenKind(CT::T_GROUP_IMPORT_BRACE_CLOSE)) {
                return;
            }
        }
```

I would like to propose this as currently the `T_GROUP_IMPORT_BRACE_CLOSE` is never found because of the priority thingy in the transformers. Checking for `&` is just a quick check for the follow test;

```php
            'return ref.' => [
                '<?php function & foo() {}',
                [
                    1 => T_FUNCTION,
                ],
            ],
```